app/Http/Controllers/MailAuthController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\AuthCode;
+use Illuminate\Http\Request;
+use App\Http\Requests\MailAuthRequest;
+use Illuminate\Support\Str;
+use Carbon\Carbon;
+use App\Mail\SendAuth;
+use Illuminate\Support\Facades\Mail;
+
+class MailAuthController extends Controller
+{
+    public function show_authcode_form()
+    {
+        return view('authcode.create');
+    }
+
+    public function create_authcode(MailAuthRequest $request)
+    {
+        $authcode = new AuthCode;
+        $authcode->authcode = Str::random(15);
+        $authcode->email = $request->email;
+        $authcode->expired_at = Carbon::now()->addMinutes(15);
+        $authcode->save();
+
+        Mail::to($authcode->email)
+            ->send(new SendAuth($authcode));
```

Comment:
ここの改行は必要ですか？
```suggestion
        Mail::to($authcode->email)->send(new SendAuth($authcode));
```
この長さであれば、改行しない方がすっきりすると思います。